include/unifex/trampoline_scheduler.hpp

```diff
@@ -134,6 +134,12 @@ class scheduler {
   schedule_sender schedule() const noexcept {
     return schedule_sender{maxRecursionDepth_};
   }
+  friend bool operator==(scheduler a, scheduler b) noexcept {
+    return a.maxRecursionDepth_ == b.maxRecursionDepth_;
+  }
+  friend bool operator!=(scheduler a, scheduler b) noexcept {
+    return a.maxRecursionDepth_ != b.maxRecursionDepth_;
+  }
```

Comment:
I wonder whether all trampoline_schedulers should be considered equal. Since they all dispatch work to the same trampoline queue in the end.

Do we have a definition of what equality of schedulers means?

Reply:
I wondered that too. I think it means: refers to the same underlying execution context, in which case these schedulers should always compare equal. I'll change it.